src/hooks/useFormValidation.ts

```
@@ -2,274 +1,72 @@
-            delete newErrors[field]
-          }
-        }
-        
-        return {
-          ...prev,
-          data: newData,
-          errors: newErrors,
-          isDirty: true,
-          isValid: Object.keys(newErrors).length === 0
-        }
+  const validateField = (field: string, value: any, schema: z.ZodSchema): string | null => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)